.eslintrc.json

```diff
@@ -56,6 +56,7 @@
     "react/jsx-indent": "off",
     "react/jsx-indent-props": "off",
     "react-hooks/rules-of-hooks": "off",
+    "prettier/prettier": "off",
```

Comment:
Why are we disabling this?

Reply:
Because we don't need to get prettier warnings/errors within `eslint`